demonstrations/tutorial_how_to_use_registers.metadata.json

```
@@ -0,0 +1,42 @@
+{
+    "title": "How to use wire registers",
+    "authors": [
+        {
+            "id": "austin_huang"
```

@austingmhuang , we need this format here:
```suggestion
            "username": "your_PennyLane_Profile_username"
```
If you don't have a profile yet, please make one on pennylane.ai. It should contain your name, an image and a headline, but adding more information is good.
If you have any questions, you can also ask me on Slack.

---

Hi @ikurecic,

I made [my profile](https://pennylane.ai/profile/austingmhuang) just now, so I'll just write in `austingmhuang` following "username":